docs/man_pages/project/creation/sample-website.md

```diff
@@ -0,0 +1,31 @@
+sample website
+==========
+
+Usage | Syntax
+------|-------
+General | `$ appbuilder sample website`    
+
+Lists all available Mobile Website sample apps with name, description, GitHub repository, and clone command. To clone a selected sample app,
```

Comment:
Lists all available mobile website sample apps with name, description, GitHub repository and clone command. To clone a selected sample app, run its clone command.